### DISCUSSION TITLE
Custom validator does not throw exception inside `updating` hook

### DISCUSSION THREAD

**Discussion by Roardom** [livewire/livewire#8499] [7. Bugs (archived)]
### Livewire version

v3.5.0

### Laravel version

v11.8.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1. Have a typed integer property
2. Use an `updating` hook on this property to validate if it is an integer or not
3. On the front-end, type `123` into the textbox. It works fine.
4. Next, type `abc`. An error 500 is generated: "Cannot assign string to property App\Livewire\Simple::$number of type int".

The validator should have thrown an exception on `abc` because it is not numeric, but the property gets set anyways.

### Repository or Wirebox URL

https://wirebox.app/b/x6dn2

### Additional information

```php
<?php

declare(strict_types=1);

namespace App\Http\Livewire;

use Illuminate\Support\Facades\Validator;
use Livewire\Component;

class TypedLivewireTest extends Component
{
    public int $number;

    final public function updatingNumber(mixed &$number): void
    {
        Validator::make(['number' => $number], [
            'number' => ['required', 'decimal:0'],
        ])->validate();

        $number = (int) $number;
    }

    final public function render(): string
    {
        return <<<'HTML'
        <div>
            <input name="number" wire:model.live="number">
            Entered: {{ $this->number ?? '' }}
        </div>
        HTML;
    }
}
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by DaniilSkLi** [comment]
Similar problem. Any `throw` livewire just ignores and I don't know how to stop it from executing except `return` in the called action.
